### DISCUSSION TITLE
Rename SG-1000 to include Othello Multivision

### DISCUSSION THREAD

**Discussion by rzumer** [RetroAchievements/RAWeb#760] [Ideas]
The Othello Multivision platform is supported and linked with SG-1000. It is a clone platform which is hardware compatible, so it should be fine to use the same entry for both, but for clarity I recommend renaming "SG-1000" to "SG-1000/Othello Multivision" in the database.

**Comment by meleu** [comment]
According to the https://segaretro.org/Othello_Multivision:

> The Othello Multivision (オセロマルチビジョン) is a licensed SG-1000 clone manufactured by Tsukuda Original.

Being it just a clone (TBF: I'm not sure if it's "just" a clone), do you think it needs to be publicized like that?

My reasoning is that we also don't publicize names like Genesis or Famicom...

I'm not very aware of the "sega retro" scene, but at a quick glance it seems that SG-1000 is clear enough.

**Comment by rzumer** [comment]
As far as I know the Multivision has "exclusive" titles that do not work on an SG-1000 while SG-1000 titles remain compatible on it, so it is not the same as a simple name change. I don't know much about it either though.

"Genesis" is included in the title in the Site Pages dropdown, but not in the database title, FYI.

**Comment by Sanaki** [comment]
I'd rather not make this change. While it's possible there are exclusives, it still is simply a clone. I wouldn't go renaming NES to include Dendy either. Perhaps if we ever add a blurb on each system describing it for the users, we can mention the clones under the same header there.

**Comment by Flashfire42** [comment]
A blurb would be nice especially with the bootlegs that are on RA as well. Hardware clones deserve a note particularly when like in the case of the Multivision there are compatibility differences.
